openapi-cli/src/test/resources/expected_gen/petstore_client_swagger.bal

```diff
@@ -0,0 +1,266 @@
+            }
+            if config.proxy is http:ProxyConfig {
+                httpClientConfig.proxy = check config.proxy.ensureType(http:ProxyConfig);
+            }
+        }
+        if config.auth is ApiKeysConfig {
+            self.apiKeyConfig = (<ApiKeysConfig>config.auth).cloneReadOnly();
+        } else {
+            config.auth = <http:BearerTokenConfig>config.auth;
+            self.apiKeyConfig = ();
+        }
+        http:Client httpEp = check new (serviceUrl, httpClientConfig);
+        self.clientEp = httpEp;
+        return;
+    }
+    # Update an existing pet
+    #
+    # + payload - Pet object that needs to be added to the store
+    # + return - Invalid ID supplied
+    remote isolated function updatePet(Pet payload) returns http:Response|error {
+        string resourcePath = string `/pet`;
+        http:Request request = new;
+        json jsonBody = check payload.cloneWithType(json);
+        request.setPayload(jsonBody, "application/json");
+        http:Response response = check self.clientEp->put(resourcePath, request);
+        return response;
+    }
+    # Add a new pet to the store
+    #
+    # + payload - Pet object that needs to be added to the store
+    # + return - Invalid input
+    remote isolated function addPet(Pet payload) returns http:Response|error {
+        string resourcePath = string `/pet`;
+        http:Request request = new;
+        json jsonBody = check payload.cloneWithType(json);
+        request.setPayload(jsonBody, "application/json");
+        http:Response response = check self.clientEp->post(resourcePath, request);
+        return response;
+    }
+    # Finds Pets by status
+    #
+    # + status - Status values that need to be considered for filter
+    # + return - successful operation
+    remote isolated function findPetsByStatus(string[] status) returns Pet[]|error {
+        string resourcePath = string `/pet/findByStatus`;
+        map<anydata> queryParam = {"status": status};
+        map<Encoding> queryParamEncoding = {"status": {style: FORM, explode: true}};
+        resourcePath = resourcePath + check getPathForQueryParam(queryParam, queryParamEncoding);
+        Pet[] response = check self.clientEp->get(resourcePath);
+        return response;
+    }
+    # Finds Pets by tags
+    #
+    # + tags - Tags to filter by
+    # + return - successful operation
+    #
+    # # Deprecated
+    @deprecated
+    remote isolated function findPetsByTags(string[] tags) returns Pet[]|error {
+        string resourcePath = string `/pet/findByTags`;
+        map<anydata> queryParam = {"tags": tags};
+        map<Encoding> queryParamEncoding = {"tags": {style: FORM, explode: true}};
+        resourcePath = resourcePath + check getPathForQueryParam(queryParam, queryParamEncoding);
+        Pet[] response = check self.clientEp->get(resourcePath);
+        return response;
+    }
+    # Find pet by ID
```

Comment:
Missing blank lines in-between remote functions in the generated client

Reply:
It is how the client is getting generated from the tool. There is no new line between remote functions.